github_connector/wizards/wizard_update_branch_list.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class WizardUpdateBranchList(models.TransientModel):
+    _name = 'wizard.update.branch.list'
```

Comment:
I get this warning:
`wizard.update.branch.list has no _description `
so I think It's better that you add the parameter
`_description`